OpenRA.Mods.Common/Activities/Hunt.cs

```diff
@@ -25,10 +25,10 @@ public class Hunt : Activity
 		public Hunt(Actor self)
 		{
 			move = self.Trait<IMove>();
-			var attack = self.Trait<AttackBase>();
+			var attacks = self.TraitsImplementing<AttackBase>();
 			targets = self.World.ActorsHavingTrait<Huntable>().Where(
```

Comment:
Not related to this PR, but i'd like to point out that wow, this is really bad for performance.  It would be good if a future PR could rework the Hunt activity to not query and check every actor on the map every tick.